GetServices/handler.ts

```diff
@@ -40,11 +43,46 @@ type IGetServicesHandler = (
   auth: IAzureApiAuthorization
 ) => Promise<IGetServicesHandlerResult>;
 
+interface IFoldableResultIterator<T> {
+  readonly executeNext: (init: T) => Promise<Either<CosmosErrors, T>>;
+}
+
+function reduceResultIterator<A, B>(
```

Comment:
can we use the one from io-fn-commons?

Reply:
Do we have a `reduceResultIterator` in io-fn-commons?

Reply:
we used to (https://github.com/pagopa/io-functions-commons/blob/v10.7.0/src/utils/documentdb.ts#L544).
it should be restored there :)

Reply:
I've used mapAsyncIterator instead of reduceAsyncIterator ( it was deleted from io-fn-commons ) @gunzip